deltachat-ffi/src/lib.rs

```diff
@@ -3235,7 +3235,7 @@ pub unsafe extern "C" fn dc_lot_get_text1(lot: *mut dc_lot_t) -> *mut libc::c_ch
     }
 
     let lot = &*lot;
-    strdup_opt(lot.get_text1())
+    lot.get_text1().strdup()
```

Comment:
I find the strdup_opt call actually more readable.  In the second form i am tempted to assume that get_text1() returns a string.  I guess i am more hesitant to add methods to existing types as a general technique.

Reply:
The beauty of this is that you don't actually have to know what .get_text1() returns, you know what strdup() returns which is what you care about.